crates/youki/src/commands/features.rs

```diff
@@ -1,8 +1,277 @@
+            String::from("suid"),
+            String::from("symfollow"),
+            String::from("sync"),
+            String::from("tmpcopyup"),
+            String::from("unbindable"),
+        ]),
+        linux: Some(Linux {
+            namespaces: Some(namespaces),
+            capabilities: Some(capabilities),
+            cgroup: Some(Cgroup {
+                v1: Some(true),
+                v2: Some(true),
+                systemd: Some(true),
+                systemd_user: Some(true),
+            }),
+            seccomp: Some(Seccomp {
+                enabled: Some(true),
+                actions: Some(vec![
+                    String::from("SCMP_ACT_ALLOW"),
+                    String::from("SCMP_ACT_ERRNO"),
+                    String::from("SCMP_ACT_KILL"),
+                    String::from("SCMP_ACT_KILL_PROCESS"),
+                    String::from("SCMP_ACT_KILL_THREAD"),
+                    String::from("SCMP_ACT_LOG"),
+                    String::from("SCMP_ACT_NOTIFY"),
+                    String::from("SCMP_ACT_TRACE"),
+                    String::from("SCMP_ACT_TRAP"),
+                ]),
+                operators: Some(vec![
+                    String::from("SCMP_CMP_EQ"),
+                    String::from("SCMP_CMP_GE"),
+                    String::from("SCMP_CMP_GT"),
+                    String::from("SCMP_CMP_LE"),
+                    String::from("SCMP_CMP_LT"),
+                    String::from("SCMP_CMP_MASKED_EQ"),
+                    String::from("SCMP_CMP_NE"),
+                ]),
+                archs: Some(vec![
+                    String::from("SCMP_ARCH_AARCH64"),
+                    String::from("SCMP_ARCH_ARM"),
+                    String::from("SCMP_ARCH_MIPS"),
+                    String::from("SCMP_ARCH_MIPS64"),
+                    String::from("SCMP_ARCH_MIPS64N32"),
+                    String::from("SCMP_ARCH_MIPSEL"),
+                    String::from("SCMP_ARCH_MIPSEL64"),
+                    String::from("SCMP_ARCH_MIPSEL64N32"),
+                    String::from("SCMP_ARCH_PPC"),
+                    String::from("SCMP_ARCH_PPC64"),
+                    String::from("SCMP_ARCH_PPC64LE"),
+                    String::from("SCMP_ARCH_RISCV64"),
+                    String::from("SCMP_ARCH_S390"),
+                    String::from("SCMP_ARCH_S390X"),
+                    String::from("SCMP_ARCH_X32"),
+                    String::from("SCMP_ARCH_X86"),
+                    String::from("SCMP_ARCH_X86_64"),
+                ]),
+            }),
+            apparmor: Some(Apparmor {
+                enabled: Some(true),
+            }),
+            selinux: Some(Selinux {
+                enabled: Some(true),
+            }),
+        }),
+        annotations: {
+            let mut annotations_map = HashMap::new();
+            annotations_map.insert(
+                ANNOTATION_LIBSECCOMP_VERSION.to_string(),
+                String::from("2.5.3"),
```

Comment:
This should be fetched from libseccomp (if youki uses libseccomp)

Reply:
it definitely does. what's the best way for me to grab this version number? I apologize, I'm pretty new to rust.